src/main/kotlin/eu/europa/ec/eudi/openid4vci/AuthorizeIssuance.kt

```diff
@@ -37,6 +39,15 @@ data class AuthorizationRequestPrepared(
     val dpopNonce: Nonce?,
 ) : java.io.Serializable
 
+@Serializable
```

Comment:
@babisRoutis I don't believe we need to mark this as JSON Serializable. Isn't this enum part of the core domain? 
Also, even though I think it's not needed for enum, please make this enum extend java.io.Serializable.

Reply:
Just noticed we use this enum is `DeferredIssuanceStoredContextTO`. Would it make sense to introduce a separate enum for serialization, and avoid Kotlin Serialization annotations in the core domain?

Reply:
They idea was to use it with Deferred Issuer.
Anyway, implemented `java.io.Serializable` removed kotlinx serializable.